### DISCUSSION TITLE
Minimum resident-count for nation creation question, re: losing residents.

### DISCUSSION THREAD

**Discussion by Zalaee** [TownyAdvanced/Towny#5174] [Q&A]
Upon setting minimum amount of residents town needs to make a nation, after it's reached and nation is made nation will automaticly delete if the capital town resident amount falls below minimum needed to create nation.
Is it possible to toggle off the automatic nation delete upon falling below minimum residents in capital town?

**Comment by LlmDl** [comment]
It is not. The setting should not be used if you don't want deletion to occur.
